docs/plugin_developers_guide/01_Fledge_plugins.rst

```diff
@@ -116,38 +115,97 @@ Required APIs include:
+
+
+Rule plugin APIs
+------------------------
+
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_eval            | Rule            | Takes JSON asset document to eval; Returns True if should "deliver" a notification                    |
 +------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
-| plugin_triggers        | Notification    | Returns JSON asset document describing what notification triggers have fired                          |
+| plugin_triggers        | Rule            | Returns JSON asset document describing what notification triggers have fired                          |
 +------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
-| plugin_reason          | Notification    | Takes JSON asset document describing why notifications have fired                                     |
+| plugin_reason          | Rule            | Takes JSON asset document describing why notifications have fired                                     |
 +------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
-| plugin_deliver         | Notification    | Takes name/notification/trigger/message strings to be sent to notification target                     |
-|                        | |br| delivery   |                                                                                                       |
+| plugin_reconfigure     | Rule            | Replaces existing configuration with new values; may need to call shutdown/init                       |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+
+
+Notification delivery APIs
+--------------------------
+
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_deliver         | Notification    | Takes name/notification/trigger/message strings to be sent to the                                     |
+|                        |                 | `notification delivery types`_ target.                                                                |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_reconfigure     | Notification    | Replaces existing configuration with new values; may need to call shutdown/init                       |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+
+Notification delivery types
```

Comment:
No, these are not types of anything they are plugins. The set of plugins is already automatically generated and documented. This just duplicates that in a way that will need manual maintenance.